### ISSUE TITLE
Make OAR read-only while we migrate the database

### ISSUE THREAD

**Issue by jwalgran** [open-apparel-registry/open-apparel-registry#2160]
**NOTE** Points for this are already covered by other Task 51 issues

### Application changes
- [x] Disable creating `DownloadLog` and `RequestLog` records
- [x] Render alternate "not available" content in place of these UI features
  - New user registration
  - All user settings  
  - Contribute
  - Facility claim wizard 
  - Claimed facility form
- [ ] Disable login 

### System changes
- [ ] Use `ALTER DATABASE openapparelregistry SET default_transaction_read_only TO on;` to make the database read-only

**Comment by TaiWilkin** [comment]
From @mariel-oar:
Two things I want to close the loop on with this:

We had discussed putting a banner at the top of the screen that messages the freeze (e.g. "Uploading new data to the OAR is disabled as of October 14 while we prepare to transition this database into Open Supply Hub. Check back on November 2 when OS Hub goes live.”)

Primary CTA: Contribute button should be greyed out

Nice to have:
3) Facility pages: CTAs for claim, report as duplicate, report as close, and suggest an edit should be removed/greyed out
This one is a nice to have because we can manage these through the info inbox as well.

**Comment by mariel-oar** [comment]
thank you @TaiWilkin. Can you confirm if Azavea is good to go with displaying the banner with the copy referenced above in addition to the CTA changes?

**Comment by TaiWilkin** [comment]
@mariel-oar, yes @mstone121 is implementing that along with the other changes.

**Comment by jwalgran** [comment]
@mstone121 Som changes to make after additional testing on staging

1. The Confirm and Reject buttons on the list details page will not work, so we should disable them. To see these buttons in development you can log in as c9@example.com and use Mu List from the user menu in the header to navigate to the list details.
![Screen Shot 2022-10-05 at 11.26.24 AM.png](https://images.zenhubusercontent.com/54e3a7217c6ccc6277a72de5/f239a8fd-c13c-4222-9cd5-7d2e5c48599a)

2. Our partners make POST requests to /api/facilities/ and they currently fail with a 500. We can add this as the verify first line of `FacilitiesViewSet.create` to be more helpful.

```python
        return Response('Uploading new data to the OAR is disabled as of '
                        'October 14 while we prepare to transition this '
                        'database into Open Supply Hub.',
                        status=status.HTTP_403_FORBIDDEN)
```

**Comment by mstone121** [comment]
@jwalgran Updates from your comment have been pushed to the read-only branch: https://github.com/open-apparel-registry/open-apparel-registry/pull/2184#commits-pushed-407a776

**Comment by jwalgran** [comment]
I ran into a problem logging in on staging.

<img width="1109" alt="Screen Shot 2022-10-06 at 10 51 39 AM" src="https://user-images.githubusercontent.com/17363/194384740-57acb5ef-1d79-42fc-b5ba-d403b2b27db1.png">

I then realized that in addition to updating the user record with the last login time, the application is also configured to use the database for session storage

This StackOverflow post talks about it, and describes a potential solution. https://stackoverflow.com/questions/49025407/in-django-1-11-how-to-allow-users-to-login-on-a-read-only-database

I have asked Mariel on Slack if we need to allow logins.

If we do, I see two options, in order of cost/complexity

- Don't make the database read-only
- Attempt to implement something similar to the Stack Overflow answer (though we cannot use file-based sessions)
